app/controllers/organisations_controller.rb

```diff
@@ -29,6 +29,18 @@ def schemes
     @filter_type = "schemes"
   end
 
+  def download_schemes_csv
+    organisation_schemes = Scheme.where(owning_organisation_id: @organisation.id)
```

Comment:
Do we also want to get parent schemes given we display them in the list? Similar to line 24?

Reply:
Ah great spot, thanks!